docs/how-to/installation/aws.txt

```diff
@@ -418,9 +404,9 @@ Agent
 
 Master Node
 ~~~~~~~~~~~
-The master node should be deployed on an EC2 instance supporting >= 2 CPUs (Intel Broadwell or later), 4GB of RAM,
+The master node should be deployed on an EC2 instance with at least 2 CPUs (Intel Broadwell or later), 4GB of RAM,
 and 100GB of disk storage. This corresponds to an EC2 t2.medium instance or better. The AMI should be the default
-Ubuntu 16.04 AMI.
+Ubuntu 18.04 AMI.
```

Comment:
question: I believe you but since this is a factual change just want to confirm this was an intentional change

Reply:
Yeah, we already support both, so I figure there's no need to be herding people to the super old one.